resources/js/Pages/Asisten.vue

```diff
@@ -11,13 +11,43 @@
         </div>
         <div class="relative w-full h-full">
           <div class="absolute w-full h-full px-8 pt-8">
-            <div class="bg-yellow-800 rounded-t-lg w-full h-full flex">
-              <div class="w-full h-auto break-words pt-16 text-center text-yellow-200 font-merri text-2xl p-8">
-                <span class="whitespace-pre-wrap">{{ currentUser.deskripsi }}</span>
+            <div class="bg-yellow-800 rounded-t-lg w-full h-full flex-col overflow-y-auto">
+              <!-- EDIT desc assistant -->
+              <div class="pt-16 px-8 w-full h-auto text-center">
+                <span class="p-3 bg-green-800 font-merri-bold text-xl cursor-pointer text-yellow-200 rounded-lg hover:bg-green-900 animation-enable"
+                      :class="'editDescBtn'"
```

Comment:
```suggestion
                <span class="p-3 bg-green-800 font-merri-bold text-xl cursor-pointer text-yellow-200 rounded-lg hover:bg-green-900 animation-enable editDescBtn"
```

This one should be like this, you should never use a reactive class `:class` for a static one, this is meant to differ between a dynamic class and a static one

Reply:
hmm i see, thank you bang, i'll modify this suggested change